build/brnd-config-develop.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "BRND sportcenter dagsprogram",
+  "icon": "",
+  "id": "01FRJPF4XATRN8PBZ35XN84PS6",
```

Comment:
The template files build/brnd* files need to be built again for the ULID change to be added.

Run build command and add the changed build/brnd* files